test/feature_discovery_test.dart

```diff
@@ -52,7 +52,7 @@ void main() {
     });
   });
 
-  group('Non-existent featureIds', () {
+  group('Non-existent feature ids', () {
```

Comment:
I adjusted this to match the spelling I used in documentation strings as this should be read literally (there are not duplicate `featureIds` because this would literally mean that there were two lists as the list is called `featureIds`).

If you want to revert that, you should probably adjust spelling in documentation comments as well for consistency.